src/test/java/com/fundynamic/d2tm/game/map/renderer/MapCellViewportRendererTest.java

```diff
@@ -23,7 +23,7 @@
     @Mock
     private Map map;
 
-    @Test(timeout = 1000)
+    @Test(timeout = 2000)
```

Comment:
It takes long (1367ms or something) on travis. I guess because its limited vm?

On Sat, Feb 21, 2015 at 12:06 AM, Arjen van der Ende
<notifications@github.com> wrote:

>> @@ -23,7 +23,7 @@
>>      @Mock
>>      private Map map;
>>  
>> -    @Test(timeout = 1000)
>> +    @Test(timeout = 2000)
> Any idea why this test takes so long to execute?
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/stefanhendriks/dune2themaker4j/pull/42/files#r25111117